src/throttler.guard.ts

```diff
@@ -20,6 +20,7 @@ export class ThrottlerGuard implements CanActivate {
   ) {}
 
   async canActivate(context: ExecutionContext): Promise<boolean> {
+    const req = context.switchToHttp().getRequest();
```

Comment:
Just a callout that we'll need to be careful of this kind of thing when we move to working with WS and GQL.

Reply:
I was already aware of it. I'll do the merge though 😂